agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -1926,10 +1926,13 @@ bool slave_thread::handle_cmdu_ap_manager_message(Socket *sd,
         }
 
         // build 1905.1 message CMDU to send to the controller
-        if (!cmdu_tx.create(0, ieee1905_1::eMessageType::TOPOLOGY_NOTIFICATION_MESSAGE)) {
+        auto cmdu_header =
+            cmdu_tx.create(0, ieee1905_1::eMessageType::TOPOLOGY_NOTIFICATION_MESSAGE);
+        if (!cmdu_header) {
             LOG(ERROR) << "cmdu creation of type TOPOLOGY_NOTIFICATION_MESSAGE, has failed";
             return false;
         }
+        cmdu_header->flags().relay_indicator = true;
 
```

Comment:
(Not for this PR) Maybe it would be better to move this code entirely to backhaul_manager. Indeed, if we just forward the notification as-is to backhaul manager, we can handle everything there. backhaul_manager anyway needs to handle the notification since it maintains a cache of the state of all radios together.

Reply:
Or... move the code from the backhaul manager to the unified Agent...